Makefile

```diff
@@ -581,15 +581,11 @@ check: ## Run prerequisite code checks
 .PHONY: check-all
 check-all: ## Check everything
 check-all: check-fmt check-clippy check-style check-markdown check-docs
-check-all: check-version check-examples check-component-features
+check-all: check-version check-examples
 check-all: check-scripts
 check-all: check-helm-lint check-helm-dependencies check-helm-snapshots
 check-all: check-kubernetes-yaml
 
-.PHONY: check-component-features
-check-component-features: ## Check that all component features are setup properly
-	${MAYBE_ENVIRONMENT_EXEC} ./scripts/check-component-features.sh
-
```

Comment:
Hmm, I will miss this feature. Could we rewrite this to use `cargo hack check --each-feature`?

Reply:
Sure, I can make that happen.

Reply:
Done. I've also revered the test workflow to refer to the make target.